bench/Autofac.BenchmarkProfiling/Autofac.BenchmarkProfiling.csproj

```
@@ -1,13 +1,13 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
```

```suggestion
    <TargetFramework>net6.0</TargetFramework>
```
We don't actually run the benchmarks in every framework - just one, enough to see if something has changed dramatically. Going with just net6 is fine.

---

Done